interface/modules/custom_modules/oe-module-claimrev-connect/public/claims.php

```diff
@@ -0,0 +1,334 @@
+    require_once '../src/ClaimSearch.php';
+    require_once '../src/ClaimRevApi.php';
+    require_once '../src/UploadEdiFileContentModel.php';
+
+   
+    use OpenEMR\Modules\ClaimRevConnector\ClaimSearch;
+    use OpenEMR\Modules\ClaimRevConnector\ClaimSearchModel;
+
+    class ClaimsPage
+    {
+        public static function SearchClaims($postData)
+        {
+            $firstName = $_POST['patFirstName']; 
+            $lastName = $_POST['patLastName']; 
+            $startDate = $_POST['startDate']; 
+            $endDate = $_POST['endDate']; 
+
+            $model = new ClaimSearchModel();
+            $model->patientFirstName = $firstName;
+            $model->patientLastName = $lastName;
+            $model->receivedDateStart = $startDate;
+            $model->receivedDateEnd = $endDate;
+
+            $data = ClaimSearch::Search($model);
+            return $data;
+        }
+    }
+?>
+<html>
+    <head>
+        <link rel="stylesheet" href="../../../../../public/assets/bootstrap/dist/css/bootstrap.min.css">
+    </head>
+    <title>ClaimRev Connect - Claims</title>
+    <body>
+        <div class="row"> 
+            <div class="col">
+            <?php
+               
+                include '../templates/navbar.php';
+            ?>
+            </div>
+        </div>
+        <form method="post" action="<?=$_SERVER['PHP_SELF'];?>">
+            <div class="card">  
+                <div class="row">
+                    <div class="col">
+                        <div class="form-group">
+                            <label for="startDate">Send Date Start</label>
+                            <input type="date" class="form-control"  id="startDate" name="startDate" value="<?php echo isset($_POST['startDate']) ? $_POST['startDate'] : '' ?>"  placeholder="yyyy-mm-dd"/>
+                        </div>
+                    </div>                    
+                    <div class="col">
+                        <div class="form-group">
+                            <label for="endDate">Send Date End</label>
+                            <input type="date" class="form-control"  id="endDate" name="endDate" value="<?php echo isset($_POST['endDate']) ? $_POST['endDate'] : '' ?>" placeholder="yyyy-mm-dd"/>
+                        </div>
+                    </div>
+                    <div class="col">
+                      
+                    </div>                    
+                    <div class="col">
+                        
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col">
+                        <div class="form-group">
+                            <label for="patFirstName">Patient First Name</label>
+                            <input type="text" class="form-control"  id="patFirstName" name="patFirstName"  value="<?php echo isset($_POST['patFirstName']) ? $_POST['patFirstName'] : '' ?>"  placeholder="Patient First Name"/>
```

Comment:
In this line will use attr() to html escape a variable in a html attribute and xla() to translate a variable in a html attribute:
```
<input type="text" class="form-control"  id="patFirstName" name="patFirstName"  value="<?php echo isset($_POST['patFirstName']) ? attr($_POST['patFirstName']) : '' ?>"  placeholder="<?php echo xla("Patient First Name"); ?>"/>
```